core/uploadDiveLogsDE.cpp

```diff
@@ -261,13 +261,14 @@ void uploadDiveLogsDE::slot_updateProgress(qint64 current, qint64 total)
 {
 	if (!reply)
 		return;
+	
 	if (total <= 0 || current <= 0)
 		return;
 
-	// Calculate percentage
+	// Calculate percentage as 0.x (values between 0.0 and 1.0)
 	// And signal whoever wants to know
 	qreal percentage = (float)current / (float)total;
-	emit uploadProgress(percentage);
+	emit uploadProgress(percentage, 1.0);
```

Comment:
why not just pass `uploadProgress(current, total)` ?

Reply:
Because then I would need to do the calculation on the QML site. The old desktop code also did not use current/total directly. Instead of having arbitrary values it makes sense to me to use percentage, the total is needed to tell if the range is 0-1 or 0-100

Reply:
Then I would need to do calculations in QML. The old desktop code also did manipulate current/total. I found it logical to use percentage, and total is there for convenience (and to signal that the range is 0-1 and not 0-100)

Reply:
makes sense. Thanks for the explanation!